remote-config/src/main/kotlin/com/rakuten/tech/mobile/remoteconfig/RemoteConfigInitProvider.kt

```diff
@@ -40,6 +40,16 @@ class RemoteConfigInitProvider : ContentProvider() {
          **/
         @MetaData(key = "com.rakuten.tech.mobile.ras.ProjectSubscriptionKey")
         fun subscriptionKey(): String
+
+        /**
+         * Delay in minutes between polls to the Config API.
+         * Set to 60 minutes by default.
+         **/
+        @MetaData(
+            key = "com.rakuten.tech.mobile.ras.PollingDelay",
+            value = "60"
```

Comment:
Is this it setting a default?

Reply:
Yeah, `default` is a reserved keyword in Java, so that's why this library uses `value` instead.